template/carousel/carousel.html

```diff
@@ -1,6 +1,6 @@
 <div ng-mouseenter="pause()" ng-mouseleave="play()" class="carousel" ng-swipe-right="prev()" ng-swipe-left="next()">
     <ol class="carousel-indicators" ng-show="slides.length > 1">
-        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
+        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="!$currentTransition && select(slide)"></li>
```

Comment:
I'm willing to make that change, but it should be noted that both the `next` and `prev` methods check that property as well, and if it is falsey, then `select` is called. Theoretically, I would prefer to have it be inside the function instead of being parsed by Angular, but then it requires thought about those methods. I could move the check into `select` instead and delete those lines from the `next` and `prev` functions.